node.gyp

```diff
@@ -250,6 +250,26 @@
             'deps/v8/src/third_party/vtune/v8vtune.gyp:v8_vtune'
           ],
         }],
+        [ 'v8_inspector=="true"', {
+          'defines': [
+            'HAVE_INSPECTOR=1',
```

Comment:
Can you add a HAVE_INSPECTOR=0 define when `v8_inspector=="false"`?  That avoids pedantic compiler warnings about HAVE_INSPECTOR being undefined.